src/equations/equations_parabolic.jl

```diff
@@ -2,11 +2,17 @@
 # specialize this function to compute gradients e.g., of primitive variables instead of conservative
 gradient_variable_transformation(::AbstractEquationsParabolic) = cons2cons
 
+# By default, the gradients are taken with respect to the conservative variables.
+# this is reflected by the type parameter `GradientVariablesConservative` in the abstract
+# type `AbstractEquationsParabolic{NDIMS, NVARS, GradientVariablesConservative}`.
+abstract type AbstractGradientVariable end
+struct GradientVariablesConservative <: AbstractGradientVariable end
```

Comment:
Why introduce `AbstractGradientVariable`? It seems to not be used for anything, so I am (honestly) wondering why to add a new supertype.

Reply:
I thought (incorrectly) that I needed an abstract type's parameter to be another abstract type...

I fixed this in the abstract type but forgot to remove this afterwards. Will fix in the morning. Thanks for catching it!